doc/api/api_changes.rst

```diff
@@ -11,6 +11,26 @@ sources of the changes you are experiencing.
 For new features that were added to matplotlib, please see
 :ref:`whats-new`.
 
+Changes in 1.5.3
+================
+
+Default Behavior Changes
+------------------------
+
+Prior to 1.5.3 kwargs passed to `~matplotlib.Axes.plot` were handled
+in two parts -- user supplied kwargs and default kwargs generated
+`~matplotlib.Axes.plot` (such as the cycled styles).  The two
+dictionaries are now merged and passed into
+`~matplotlib.lines.Line2D.__init__` all at once instead of applying the
+user supplied kwargs via a bulk update after the artist has been created.
+
+This causes ``None`` passed as ``plot(..., marker=None)`` to now 'do
+the default' thing rather than to result in no marker.  This is only
+an issue if the user explicitly passes ``None`` and has changed the
```

Comment:
"the user" seems like a different PoV than the rest of this document, though it's not perfectly consistent anyway...